frontend/src/pages/OrganizationAddPage.tsx

```diff
@@ -60,10 +60,12 @@ export const OrganizationAddPage = () => {
           <Typography variant="h1">
             <Title>Add Organization Members</Title>
           </Typography>
-          <ContactSelector
-            contacts={contacts.filter(c => !members.find(s => s.user.id === c.id))}
-            onChange={setEmails}
-          />
+          <Gutters top={null}>
+            <ContactSelector
+              contacts={contacts.filter(c => !members.find(s => s.user.id === c.id))}
```

Comment:
maybe move the filter before of the return. what do you think?

Reply:
Sounds good!